locator/tablets.cc

```diff
@@ -321,6 +321,42 @@ size_t tablet_map::external_memory_usage() const {
     return result;
 }
 
+bool resize_decision::operator==(const resize_decision& o) const {
+    return way.index() == o.way.index() && phase == o.phase;
+}
+
+std::strong_ordering resize_decision::operator<=>(const resize_decision& other) const {
+    return phase <=> other.phase;
+}
```

Comment:
It's not accurate to say that two resize decisions with the same timestamp and different decisions are the same. If you want to compare the timestamps, compare the timestamps, but don't claim you're comparing the decision.

Reply:
And notice how it's inconsistent with operator==.

Reply:
apparently I don't even use it. and it would only be used for determining precedence.

Reply:
akin to sstable::compare_by_max_timestamp()